custom/icds/tasks/sms.py

```diff
@@ -51,3 +51,35 @@ def send_monthly_sms_report():
         send_html_email_async.delay(subject, recipients, message,
                                     email_from=settings.DEFAULT_FROM_EMAIL)
         raise e
+
+
+@task
+def send_custom_sms_report(start_date, end_date):
+    subject = _('Monthly SMS report')
+    recipients = ['ayogi@dimagi.com', 'akaul@dimagi-associate.com', 'smazumdar@dimagi.com',
+                  'stewari@dimagi.com', 'pgoyal@dimagi.com', 'asharma@dimagi.com']
+    filename = call_command('get_icds_sms_usage', 'icds-cas', str(start_date), str(end_date))
+    try:
+        with open(filename, 'rb') as f:
+            cached_download = expose_cached_download(
+                f.read(), expiry=24 * 60 * 60, file_extension=file_extention_from_filename(filename),
+                mimetype=Format.from_format(Format.XLS_2007).mimetype,
+                content_disposition='attachment; filename="%s"' % filename)
+            path = reverse('retrieve_download', kwargs={'download_id': cached_download.download_id})
+            link = f"{web.get_url_base()}{path}?get_file"
+            message = _("""
+            Hi,
+            Please download the sms report for last month at {link}.
+            The report is available only till midnight today.
```

Comment:
I think this would need to say "next 24 hours". Midnight made sense when we generated the report around midnight so it was valid for 24 hours, hence next midnight

Reply:
Done